example/src/main/config/hono-service-messaging-config-enmasse.yml

```diff
@@ -0,0 +1,32 @@
+hono:
+  app:
+    maxInstances: 1
+    healthCheckBindAddress: 0.0.0.0
+    healthCheckPort: 8088
+  auth:
+    host: ${hono.auth.host}
+    port: 5671
+    keyPath: /run/secrets/hono-messaging-key.pem
+    certPath: /run/secrets/hono-messaging-cert.pem
+    trustStorePath: /run/secrets/trusted-certs.pem
+    name: 'Hono Messaging'
+    validation:
+      certPath: ${hono.auth.validation.certPath}
+  downstream:
+    host: messaging.hono
```

Comment:
is this service name pre-defined by enMasse deployment or can we change this to e.g. enmasse.hono? Using `messaging.hono` is a little confusing because this is the config file for the *Hono Messaging* service and thus it sounds like we would connect to *ourselves*.

Reply:
It's predefined by enmasse. I'm not sure if there's a way to change it. Will take a look

Reply:
@dejanb  Isn't it the "messaging" OpenShift service you are talking about ? it's just "messaging" and not "messaging.hono".

Reply:
if it is deployed to the *hono* project (`-n hono`) then you can refer to it by either just *messaging* (when referring from inside *hono*) or by explicitly qualifying it with the project name when outside of *hono*.

Reply:
I think that for simplicity we can deploy it inside hono project and referring as messaging but messaging.hono sounds strange to me.

Reply:
We are referring to all services including the `.hono` suffix. I think we should be consistent and use it here as well (if we deploy enMasse to the *hono* project, which we are not required to do, are we?)

Reply:
We are not required to deploy EnMasse to the hono project, but to me it looks like a good solution at the moment. Maybe we should change our files so that hono can be deployed to the arbitrary project, so that services are referred to with the appropriate prefix?